src/column.ts

```diff
@@ -120,21 +120,27 @@ export class ColumnFactory {
 
 export class RawColumn {
     public cellLength: number = 0;
-    constructor(public columnValues: string[]) { 
+    public cellLengths: number[] = [];

```

Comment:
Becuase in other place need to compute the every col's length, so I cached in there.